app/services/description_validator.rb

```diff
@@ -21,4 +31,12 @@ def valid?
   def duplicate_headers
     @headers.group_by { |e| e }.filter { |_k, v| v.count > 1 }.keys
   end
+
+  def duplicate_druids
+    @csv.map { |row| row['druid'] }.group_by { |e| e }.filter { |_k, v| v.count > 1 }.keys
+  end
+
+  def required_headers
+    %w[druid]
+  end
```

Comment:
this is in anticipation that #3439 will allow for simple header validation and that we can just expand this array to do that ticket ... if not and we are unlikely to need further additional required headers, we could simply this 

Suggest we wait for clarification on #3439 before changing